robotiq_force_torque_sensor/launch/ur5_fts_test.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<launch>
+    <param name="robot_description" command="$(find xacro)/xacro.py '$(find robotiq_force_torque_sensor)/urdf/ur5_fts_test.urdf.xacro'" />
+
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+
+    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
+        <param name="use_gui" value="true" />
+    </node>
+
+    <!-- rviz -->
+    <node pkg="rviz" type="rviz" name="rviz"/>
+</launch>
```

Comment:
Is this file necessary? It adds a dependency (through `ur5_fts_test.urdf.xacro`) `ur_description`.

Seems strange to make the `robotiq` pkg depend on an arbitrary robot vendor support pkg?